data/ga/legislature/Danny-Mathis-0db72b1b-2be3-4596-81b2-0bdd733c9bce.yml

```diff
@@ -0,0 +1,38 @@
+id: ocd-person/0db72b1b-2be3-4596-81b2-0bdd733c9bce
+name: Danny Mathis
+given_name: Danny
+family_name: Mathis
+email: danny.mathis@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/mathis-danny-4959.jpg
```

Comment:
@sroomf this person served in GA House District 144 from January 2019 until 01/09/2023 when they began serving in district 149. I don't see their original file in this pull but I do see they show up on openstates. https://openstates.org/search/?query=danny+mathis&state=ga